packages/util/src/ssz.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  ByteVectorType,
+  ContainerType,
+  ListCompositeType,
+  UintBigintType,
+  UintNumberType,
+} from '@chainsafe/ssz'
+
+import { MAX_WITHDRAWALS_PER_PAYLOAD } from './constants'
+
+export const UintNum64 = new UintNumberType(8)
+export const UintBn64 = new UintBigintType(8)
```

Comment:
Can we changes this to `UintBigInt64`?  I don't like `BN` references.

Reply:
ok sounds good :+1: